dts/arm/nordic/nrf9160.dtsi

```diff
@@ -37,12 +37,20 @@
 		uart-2 = &uart2;
 		uart-3 = &uart3;
 		adc-0 = &adc;
+		egu-0 = &egu0;
```

Comment:
Do we really need to add these aliases?  Can we not use DT_INST defines?

Reply:
No, we can't use `DT_INST_*` macros, see my [comment](https://github.com/zephyrproject-rtos/zephyr/pull/18937#issuecomment-528838585) above.

Reply:
Than, lets not add new aliases.

Reply:
Sorry, misread your comment.

Reply:
If #18986 gets approved for work and resolved then we can remove them.